### PR TITLE
Add bash style guide

### PR DESCRIPTION
**Pull request by damacus**

### CODE REVIEW

File: guides/shell_style.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Do Not Use
+
+These are syntax or stlye recommendations to *avoid*
+
+### `set -x`
+
+The use of `-x` makes output, espeically in CI unreadable. It is only recommended for use when debugging.
+
+## Set on the shebang
```
Comment (hansnqyr) [inline guides/shell_style.md]: This indentation is a bit off

File: guides/shell_style.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+category: Bash Style
+---
+# Bash Style
+
+Where possible, we want to be consistent with our use of Bash so engineers switching betwen projects
```
Comment (hansnqyr) [inline guides/shell_style.md]: this line just trails o...

Comment (damacus) [reply guides/shell_style.md]: Indeed, it's not really finished but i wanted to get some feedback on the document as a whole :)

File: guides/shell_style.md (hunk 1 above)
Comment (damacus) [reply guides/shell_style.md]: Is it? :/

Comment (hansnqyr) [reply guides/shell_style.md]: yeah - it puts shebang, do, don't all at same level

**Pull request closed without merging**